### PR TITLE
[macOS] Software Updates 2022 Week 33

### PR DESCRIPTION
**Pull request by al-cheb**
# Description
- Xamarin.IOS 15.12.0.2
- Xamarin.Mac 8.12.0.2
- Xamarin.Android 13.0.0.0, 12.3.3.3
- Set default bundle to 6_12_18

#### Related issue:
https://github.com/actions/runner-images/issues/6038

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by mikhailkoliada — APPROVED** [review]

**Review by ddobranic — APPROVED** [review]

**Review by igorboskovic3 — APPROVED** [review]

File: images/macos/toolsets/toolset-12.json [hunk 1]
```diff
@@ -19,16 +19,30 @@
...
+            "15.12.0.2", "15.10.0.5", "15.8.0.3", "15.6.0.3", "15.4.0.0", "15.2.0.17", "15.0.0.6"
         ],
         "mac-versions": [
-            "8.10.0.5", "8.8.0.3", "8.6.0.3", "8.4.0.0", "8.2.0.17", "7.14.0.27"
+            "8.12.0.2", "8.10.0.5", "8.8.0.3", "8.6.0.3", "8.4.0.0", "8.2.0.17", "7.14.0.27"
         ],
         "android-versions": [
-            "12.2.8.3", "12.1.0.2", "12.0.0.3", "11.3.0.4"
+            "13.0.0.0", "12.3.3.3", "12.2.8.3", "12.1.0.2", "12.0.0.3", "11.3.0.4"
         ],
```
Comment (jonathanpeppers) [inline images/macos/toolsets/toolset-12.json:29]: These version numbers look right to me. How are the downloaded and installed on the image?

Comment (al-cheb) [reply images/macos/toolsets/toolset-12.json:29]: We use script to install Xamarin - https://github.com/actions/runner-images/blob/main/images/macos/provision/core/xamarin.sh and helper functions - https://github.com/actions/runner-images/blob/main/images/macos/provision/utils/xamarin-utils.sh

Comment (jonathanpeppers) [reply images/macos/toolsets/toolset-12.json:29]: Ok, great.

What decides what version of Xamarin.Android this symlink points to?

    /Library/Frameworks/Xamarin.Android.framework/Versions/Current

Just whichever one was installed *last*?

Comment (al-cheb) [reply images/macos/toolsets/toolset-12.json:29]: @jonathanpeppers , Do you think we should set up the last 6_12_19 bundle as the default?

#### Xamarin bundles
| symlink           | Xamarin.Mono | Xamarin.iOS | Xamarin.Mac | Xamarin.Android |
| ----------------- | ------------ | ----------- | ----------- | --------------- |
| 6_12_19           | 6.12         | 15.12       | 8.12        | 13.0            |
| 6_12_18 (default) | 6.12         | 15.10       | 8.10        | 12.3            |

Comment (jonathanpeppers) [reply images/macos/toolsets/toolset-12.json:29]: Yes, I would go ahead and make the latest the default every time you update these. Mobile platforms move so fast, we likely have fixes that are needed for latest Android SDK, NDK, Xcode, etc.

Comment (al-cheb) [reply images/macos/toolsets/toolset-12.json:29]: @jonathanpeppers, Thank you, updated.

**Review by jonathanpeppers — APPROVED** [review]

**Review by ddobranic — APPROVED** [review]

**Pull request merged**